tools/crm_mon.c

```diff
@@ -2017,7 +2017,7 @@ mon_refresh_display(gpointer user_data)
         return G_SOURCE_REMOVE;
     }
 
-    if (options.daemonize) {
+    if (options.daemonize || (output_format == mon_output_html && out->dest != stdout)) {
```

Comment:
Do this need to have the same criteria as the reset?

Reply:
It looks like both conditionals are safe to just be `if (out->dest != stdout)`.